process/slack_new_member_welcome.md

```diff
@@ -0,0 +1,29 @@
+<H1><img align="left" height=100 src="images/Girlgeek_w_workmark.png"><br> 
+  <em>Hello!</em></H1>
+
+&nbsp;  
+
+
+You're welcome to the Slack Platform of theGlobal PyLadies Community. Kindly do well to introduce yourself, the chapter you belong to and tell us about your involvement with the Python Programming Language.  
+
+
+
+Whiles you can make posts on this platform, you are cautioned to put them in the right channel. The current available channels are:  
+
+\#conferences
+
+\#introductions
+
+\#general
+
+\#pyladies-support
+
+\#africa
+
+
+If you are a PyLadies Organizer kindly do well to ask to be added to the organizer's channel.
+
+PyLadies aims to provide a friendly support network for women and a bridge to the larger Python world. 
```

Comment:
*Edit:*

Reword to: `PyLadies aims to provide a friendly support network for our self-identifying members and help build a bridge to the larger Python world. `